user-data.yaml

```diff
@@ -72,22 +72,20 @@ write_files:
 
     /usr/local/bin/docker-compose -f /etc/docker-compose.yml up -d
 
+- path: /usr/local/bin/setup-cdagent.sh
+  permissions: '0755'
+  content: |
+    #!/bin/bash
+     
+    aws s3 cp s3://aws-codedeploy-eu-west-1/latest/install /tmp/install --region eu-west-1
+    chmod +x /tmp/install
+    /tmp/install auto
+
 runcmd:
   - [ service, docker, start ]
-  - [ mkdir, -p, /srv/presentation ]
-  - [ mkdir, -p, /srv/mint ]
```

Comment:
No - I was under the impression that CodeDeploy should do it. 
If it runs as root then it should be able to create it.

However, it can be added back if needed.